app/app/dashboard/DashboardView.tsx

```diff
@@ -0,0 +1,151 @@
+
+        if (memberDetail.pool.includes('/')) { // Savers
+          const userSaversPercentage = BigNumber(memberDetail.liquidityUnits).div(pool.saversUnits)
+          const assetSaverToRedeem = baseAmount(BigNumber(pool.saversDepth).times(userSaversPercentage))
+          const totalRedeemValueInUsd =  baseToAsset(assetSaverToRedeem).times(pool.assetPriceUSD)
+          
+          const depositSaverValueAssetInUsd = baseToAsset(baseAmount(memberDetail.assetAdded).minus(memberDetail.assetWithdrawn)).times(pool.assetPriceUSD)
+
+          totalAddedValueInUsd = depositSaverValueAssetInUsd
+          gainInUsd = totalRedeemValueInUsd.minus(totalAddedValueInUsd)
+        } else {  // Liquidity providers
+          const userPoolPercentage = BigNumber(memberDetail.liquidityUnits).div(pool.units)
+          const assetToRedeem = baseAmount(BigNumber(pool.assetDepth).times(userPoolPercentage))
+          const runeToRedeem = baseAmount(BigNumber(pool.runeDepth).times(userPoolPercentage))
+          const redeemValueAssetInUsd =  baseToAsset(assetToRedeem).times(pool.assetPriceUSD)
+          const redeemValueAssetInRune = baseToAsset(runeToRedeem).div(pool.assetPrice).times(pool.assetPriceUSD)
+          const totalRedeemValueInUsd = redeemValueAssetInUsd.plus(redeemValueAssetInRune)
+  
+          const depositValueAssetInUsd = baseToAsset(baseAmount(memberDetail.assetAdded).minus(memberDetail.assetWithdrawn)).times(pool.assetPriceUSD)
+          const depositValueRuneInUsd = baseToAsset(baseAmount(memberDetail.runeAdded).minus(memberDetail.runeWithdrawn)).div(pool.assetPrice).times(pool.assetPriceUSD)
+          totalAddedValueInUsd = depositValueAssetInUsd.plus(depositValueRuneInUsd)
+          gainInUsd = totalRedeemValueInUsd.minus(totalAddedValueInUsd)
+        } 
+
+        const isDlp = memberDetail.runeAdded !== '0' && memberDetail.assetDeposit !== '0'
+
+        return {
+          assetId: memberDetail.pool,
+          type: memberDetail.pool.includes('/') ? PositionType.SAVER : isDlp ? PositionType.DLP : PositionType.SLP,
+          deposit: {
+            usd: totalAddedValueInUsd.amount().toNumber(),
+          },
+          gain: {
+            usd: gainInUsd.amount().toNumber(),
+            percentage: gainInUsd
+              .div(totalAddedValueInUsd)
+              .times(100)
+              .amount()
+              .toFixed(4)
+          }
+        }
+      })
+      return { pools, positions }
+    }
+  })
+
+  const totalValue = data?.positions?.reduce((total, position) => {
+    return total + position.deposit.usd + position.gain.usd;
+  }, 0);
+
+  const totalGain = data?.positions?.reduce((total, position) => {
+    return total + position.gain.usd;
+  }, 0);
+
+  const titleStyle = 'my-2 md:mb-4 md:mt-0 md:text-2xl font-medium md:mb-6 text-neutral-900 md:text-neutral font-gt-america-ext uppercase'
+  return (
+    <main className="md:mx-16">
+      <div className="flex flex-col md:mb-10">
```

Comment:
i don't think `md:mb-10` is working. There should be some space between the two sections.